recipes/replxx/all/conanfile.py

```diff
@@ -0,0 +1,64 @@
+import os
+from conans import CMake, ConanFile, tools
+
+
+class ReplxxConan(ConanFile):
+    name = "replxx"
+    description = """
+    A readline and libedit replacement that supports UTF-8,
+    syntax highlighting, hints and Windows and is BSD licensed.
+    """
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/AmokHuginnsson/replxx"
+    topics = ("conan", "readline", "libedit", "UTF-8")
+    license = "BSD"
```

Comment:
```suggestion
    license = "BSD-3-Clause"
```

Reply:
Actually the licence file has multiple licenses

Reply:
What to do here?